packages/peregrine/lib/targets/TalonWrapperConfig.js

```diff
@@ -33,6 +33,52 @@ class TalonWrapperConfig {
              */
             useApp: new WrappableTalon(addTransforms, 'App/useApp')
         };
+        /**
+         * Provides access to the talon used in Venia's Checkout UI component.
+         * @instance
+         */
+        this.Checkout = {
```

Comment:
These are legacy checkout components/talons. Should we indicate such in the property name?

For example, `this.LegacyCheckout`? Or are these property names meant to literally be the name of the path to the talon such as in `Checkout/useFlow`?

@zetlen I remember we had a discussion around naming but I don't remember this specific point.

Reply:
> Please run `prettier` and rebase!

The latest commit should have the `prettier` results